script/featured-data.js

```diff
@@ -0,0 +1,96 @@
+const featuredMusicians = [{
+  name: 'Aster Aweke',
+  image: './assets/features/aster-awoke.jpg',
+  title: 'Super Star, singer, song writer and performer',
+  detail: 'Aster Aweke is an Ethiopian Super Star, singer, song writer and performer. She is known as the queen of pop music in Ethiopia.',
+}, {
+  name: 'Teddy Afro',
+  image: './assets/features/teddy.jpg',
+  title: 'An Ethiopian singer-songwriter',
+  detail: 'Tewodros Kassahun Germamo known professionally as Teddy Afro. Known by his revolutionary songs and political dissent sentiment,most significant Ethiopian artists.',
+}, {
+  name: 'Mulatu Astatke',
+  image: './assets/features/mulatu.jpg',
+  title: 'Father of "Ethio-jazz"',
+  detail: 'An Ethiopian musician and arranger considered as the father of "Ethio-jazz".Trained in London, New York City, and Boston where he combined his jazz and Latin music  with traditional Ethiopian music',
+}, {
+  name: 'Neway Debebe',
+  image: './assets/features/neway.jpg',
+  title: 'Singer and songwriter',
+  detail: 'An Ethiopian singer and songwriter. Neway became popular after releasing his first debut album, recorded by the Tango Music shop, in 1970s with a local band called the Roha Band.',
+}, {
+  name: 'Rophnan Nuri Muzeyin',
+  image: './assets/features/rofnan.jpg',
+  title: 'Musician, DJ, and Audio Engineer',
+  detail: 'is an Ethiopian musician, singer, songwriter, DJ, and audio engineer. Widely acclaimed, he is currently considered as one of the most influential artists in Ethiopia.',
+}, {
+  name: 'Fikeraddis Nekatibeb',
+  image: './assets/features/fikre-addis.jpg',
+  title: 'Iconic female singer',
+  detail: 'A renowned iconic female singer who has actively dominated the Ethiopian music industry for the last 3 decades. She has received several national awards and international recognitions.',
+}];
+
+const musiciansList = document.querySelector('.featured-musicians-list');
+const more = document.querySelector('.more-btn');
+const less = document.querySelector('.less-btn');
+
+const speaker = function () {
```

Comment:
- [Optional] This is not the right way to name a function also when I open your file I get this 👉🏾  `Unexpected unnamed function.` Either you just declare your function like this `const speaker = () => {...}` or ` function speaker() {...}` 🙃